man/subscription-manager.8

```diff
@@ -58,6 +58,10 @@ command.
 .B subscription-manager
 has specific options available for each command, depending on what operation is being performed. Subscription Manager commands are related to the different subscription operations:
 
+.PP
+.B Note:
+Please note that using commands that requires providing a username, password, organization, or environments will require them to be passed as system arguments in a non-interactive session.
```

Comment:
i'd mention explicitly what are the affected options, e.g. "... providing an username using .B --username etc" (take care of the groff formatting); this makes it easier to see them, and to look/search for them